setup/hu.bme.mit.massif.setup/Massif.setup

```diff
@@ -578,7 +578,7 @@
             versionRange="[2.9.0,3.0.0)"/>
```

Comment:
Did you check whether any other changes are necessary in the Oomph setup (on the master stream)?

Reply:
No, not really (yet) - is it sufficient to go through the install process using the installer to see if any further changes are needed?

Reply:
Yes.